doc/sandbox.rst

```diff
@@ -0,0 +1,61 @@
+Sandbox Structure
+-----------------
+
+A sandbox is a collection of source code repositories which can be used to build one or
+more applications.
+
+Definitions
+^^^^^^^^^^^
+
+There are three types of top-level directories that are present in a sandbox.
+
+Application
```

Comment:
possibly a good place to introduce the naming conventions of sw_, lib_, app_ etc?